packages/foreman/foreman/foreman.spec

```diff
@@ -7,7 +7,7 @@
 %global release 1
 
 Name:    foreman
-Version: 3.9.1
+Version: 3.9.3
```

Comment:
Needs npm deps regeneration

Reply:
Has been done :D

Reply:
But with the wrong package.json, this looks like 3.10 or develop

Reply:
Let's call it a failed cargo-cult copy paste attempt....

`./update-requirements --excludefile ../foreman/package-exclude.json  npm ../foreman/package.json ./packages/foreman/foreman/foreman.spec`

My brain didn't notice the `../foreman/` stuff.  Fixing.